packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -311,7 +333,7 @@ contract Deploy is Deployer {
         dsi.set(dsi.recommendedProtocolVersion.selector, ProtocolVersion.wrap(cfg.recommendedProtocolVersion()));
 
         // Run the deployment script.
-        deploySuperchain.run(dsi, dso);
+        ds.run(dsi, dso);
```

Comment:
When I renamed `setupSuperchain` to `deploySuperchain` for consistency, I had a shadowed declaration warning.